### ISSUE TITLE
[Icons] Resolving Icon Alignment Issues (2 of 2)

### ISSUE THREAD

**Issue by solygambas** [adevinta/spark-tokens#68]
After #67, this follow-up pull request addresses alignment issues in several SVG icons to ensure consistent centering. Several icons are affected by misalignments and imperfect roundness, which impacts their visual presentation. 

This second batch will be based on @SimonMarquis's list:

1. [Block.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/Block.svg)
2. [Arrival.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/Arrival.svg)
3. [Brand.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/Brand.svg)
4. [Breakfast.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/Breakfast.svg)
5. [DeleteFill.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/DeleteFill.svg)
6. [DeleteOutline.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/DeleteOutline.svg)
7. [Family.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/Family.svg)
8. [GraphFill.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/GraphFill.svg)
9. [GraphOutline.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/GraphOutline.svg)
10. [ImageFill.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/ImageFill.svg)
11. [ImageOutline.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/ImageOutline.svg)
12. [Model.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/Model.svg)
13. [MoreMenuVertical.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/MoreMenuVertical.svg)
14. [OfferType.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/OfferType.svg)
15. [ParkingEstate.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/ParkingEstate.svg)
16. [Parquet.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/Parquet.svg)
17. [QuestionFill.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/QuestionFill.svg)
18. [QuestionOutline.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/QuestionOutline.svg)
19. [Salary.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/Salary.svg)
20. [SaleType.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/SaleType.svg)
21. [SeaView.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/SeaView.svg)
22. [SunOrientation.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/SunOrientation.svg)
23. [Tattoo.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/Tattoo.svg)
24. [Time.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/Time.svg)
25. [VoiceFill.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/VoiceFill.svg)
26. [VoiceOffFill.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/VoiceOffFill.svg)

We will also update the `Donate` and `Twitter` icons.

**Comment by Bibisirius** [comment]
1.[block.svg](https://www.figma.com/file/ezqkDE3LqYrek9MvYnr4dR/Spark-Foundations?type=design&node-id=316-8102&mode=design&t=HaJQUL9BkHcRuEdr-11) : 👍 
2. arrival.svg : It's the same as criteria/holidays/start.svg
I'm going to ask the designers about this
3.Brand.svg: I can't find it in the list, we have :car-brand.svg, clothes-brand.svg and mulmedia-brand.svg
4.[breakfast.svg](https://www.figma.com/file/ezqkDE3LqYrek9MvYnr4dR/Spark-Foundations?type=design&node-id=316-8102&mode=design&t=HaJQUL9BkHcRuEdr-11) 👍 
[5. DeleteFill.svg 👍 
6. DeleteOutline.svg 👍](https://www.figma.com/file/ezqkDE3LqYrek9MvYnr4dR/Spark-Foundations?type=design&node-id=316-8102&mode=design&t=HaJQUL9BkHcRuEdr-11) 
7.[Family.svg](https://www.figma.com/file/ezqkDE3LqYrek9MvYnr4dR/Spark-Foundations?type=design&node-id=316-8102&mode=design&t=HaJQUL9BkHcRuEdr-11) 👍 
[8. GraphFill.svg 👍 
9. GraphOutline.svg] 👍 (https://www.figma.com/file/ezqkDE3LqYrek9MvYnr4dR/Spark-Foundations?type=design&node-id=316-8102&mode=design&t=HaJQUL9BkHcRuEdr-11)
[10. ImageFill.svg 👍 
11. ImageOutline.svg] 👍 (https://www.figma.com/file/ezqkDE3LqYrek9MvYnr4dR/Spark-Foundations?type=design&node-id=316-8102&mode=design&t=HaJQUL9BkHcRuEdr-11)
12. [Model.svg] 👍 (https://www.figma.com/file/ezqkDE3LqYrek9MvYnr4dR/Spark-Foundations?type=design&node-id=316-8102&mode=design&t=HaJQUL9BkHcRuEdr-11)

**Comment by solygambas** [comment]
Thank you! The 'Brand' icon is indeed present in the 'multimedia' category. It's a duplicate of 'multimedia-model,' as mentioned in #71. You can locate it here: [Brand.svg](https://github.com/adevinta/spark-tokens/blob/main/assets/spark/icons/Brand.svg).
